pkgs/development/libraries/kerberos/heimdal.nix

```diff
@@ -23,6 +23,7 @@
 , CoreFoundation
 , Security
 , SystemConfiguration
+, libresolv
```

Comment:
We don't want to do those inherits anymore to make things more pkgs-by-name friendly.

Reply:
@SuperSandro2000 happy to oblige. Do you have a link to read up on the pkgs-by-name consideration? I am new to this, so I'd like to understand what it means so that I don't make a similar mistake in the future. Thanks.

Reply:
I updated as requested, but I'd still love to understand why the first version was not optimal.